client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -219,8 +218,9 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 	);
 
 	const isPluginBundleEligible = useIsPluginBundleEligible();
-	const themeDetails = useThemeDetails( selectedDesign?.slug || '' );
-	const hasThemeSoftwareSet = themeDetails?.data?.taxonomies?.theme_software_set?.length;
+	const hasThemeSoftwareSet = ( selectedDesign?.software_sets || [] ).some(
```

Comment:
Suuper nit: as the condition is checking for `woo-on-plans`, maybe call this something like `isBundledWithWooCommerce`?